scripts/ai.sh

```
@@ -0,0 +1,692 @@
+    )
+}
+
+### Function to make request to Gemini API
+# $1: prompt text
+# Returns: AI response text
+function call_gemini_api {
+    # Set trap to clear sensitive variables on exit/interrupt
+    trap 'clear_sensitive_vars' EXIT INT TERM
+    
+    local prompt="$1"
+    local api_key=$(get_ai_api_key)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)